src/main.js

```diff
@@ -1,6 +1,7 @@
-import './style.css';
-
+//import './style.css'; //Ten import wyrzuca błąd
```

Comment:
hmm i co teraz? Co to za błąd?

Reply:
Style pobierałem za pomoca taga link ale naprawiłem już ten błąd z importem więc już się pobiera przez js. (nie działa w ten sposób liveserver jak coś ;)). Błąd dotyczył tego że był importowany plik css do pliku js bez informacji że to inny rodzaj pliku niż js.